FtpClient/FtpCli/ConsoleParser/ConsoleParserCommand.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections.Generic;
+
+namespace ConsoleParser
+{
+    public class ConsoleParserCommand
+    {
+        public string commandName { get; }
+        private string helpText { get; set; }
+        private Action<List<string>> manyArgsAction;
+        private Action<string> singleArgAction;
+
+        public ConsoleParserCommand(string commandName, string helpText, Action<List<string>> action)
+        {
+            this.commandName = commandName;
+            this.helpText = helpText;
+            this.manyArgsAction = action;
```

Comment:
Should `this.manyArgsAction` be initialized and copied into? Example:

`this.manyArgsAction = new List<string>();`
`action.CopyTo(this.manyArgsAction);`